Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -65,20 +60,21 @@ protected async Task OnTextChangedHandler( string text )
             SelectedText = CurrentSearch;
             dirtyFilter = true;
 
-            if ( text?.Length >= MinLength && FilteredData.Any() )
```

Comment:
When I now set `MinLength="0"` it doesn't work as before. It must show a dropdown when a search is focused.

Reply:
My bad... I had removed this:
![image](https://user-images.githubusercontent.com/22283161/124357050-5c897880-dc11-11eb-9cfb-82b255563a41.png)

Reply:
Now it works only the first time you focus on search. If you select a value, then clear, lose focus and enter focus it will not show the menu again.

Reply:
Fixed.